modules/test/playwright/tests/object-web/objectLayout.spec.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {editObjectDefinitionPagesTest} from '../../fixtures/editObjectDefinitionPagesTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {objectPagesTest} from '../../fixtures/objectPagesTest';
+import getRandomString from '../../utils/getRandomString';
+import getRandomObjectField from './utils/getRandomObjectField';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	editObjectDefinitionPagesTest,
+	featureFlagsTest({
+		'LPS-187854': true,
+	}),
+	loginTest(),
+	objectPagesTest
+);
+
+test.describe('Manage Object Layouts through the Object Layout tab', () => {
+	test('Can view all fields of an object when creating its layout', async ({
+		apiHelpers,
+		editObjectDefinitionPage,
+		page,
+		viewObjectDefinitionsPage,
+		viewObjectLayoutsPage,
+	}) => {
+		const fields = getRandomObjectField({fieldsQuantity: 20});
```

Comment:
Hey @vngoliveira, could you use object prefix here? Also, could you check if we can use `let` instead a `const` here?

Reply:
Hey @vngoliveira, were you able to check if we can use `let` here? Since we haven't reassigned any values ​​to `objectFields`, we can probably go with a let.